src/conv.jl

```diff
@@ -288,11 +304,19 @@ for (front_name, backend, signature) in (
                                 C_in = channels_in(cdims) ÷ groupcount(cdims),
                                 C_out = channels_out(cdims) ÷ groupcount(cdims))
 
-            Threads.@sync for (wc, xc, yc) in zip(dw_cs, x_cs, dy_cs)
+            function do_work(wc, xc, yc)
                 x = @view in1[ntuple(i -> i == 4 ? xc : Colon(), 5)...]
                 dy = @view in2[ntuple(i -> i == 4 ? yc : Colon(), 5)...]
-                dw = @view out[ntuple(i -> i == 5 ? yc : Colon(), 5)...]
-                Threads.@spawn $(Symbol("$(front_name)_$(backend)!"))(dw, x, dy, cdims2; kwargs...)
+                dw = @view out[ntuple(i -> i == 5 ? yc : Colon(), 5)...] # TODO: Is this supposed to use wc?
```

Comment:
I noticed that `yc` probably should be `wc` here?

Reply:
I agree. Just to be sure, can you test that `length(dy_cs) >= length(dw_cs)` holds? Maybe via a temporary test assert. I'm pretty sure it does, but the conv code is convoluted enough that it might not.

Reply:
Added

Reply:
Sorry, I meant checking the length of the partitioned iterators before the loop. i.e. `length(dy_cs) >= length(dw_cs) || error("Inconsistent dimensions") # TODO: remove this check`.

Reply:
Ah yes. Fixed